frontend/app/services/GuardianContentService.scala

```diff
@@ -123,6 +130,20 @@ trait GuardianContent {
     }
   }
 
+  def membershipFrontContentQuery(page: Int): Future[ItemResponse] = {
+    val itemQuery = ItemQuery("/membership")
+      .showFields("all")
+      .showElements("all")
+      .pageSize(50)
```

Comment:
Yep, don't think I needed `showFields` at all, just wanted to filter by a tag. Limited `showElements` to image and reduced the page size.